solidity/contracts/BurntPixArchives.sol

```diff
@@ -91,6 +92,7 @@ contract BurntPixArchives is LSP8CappedSupply {
                 uint256(5_000)
             ));
             _setData(0x580d62ad353782eca17b89e5900e7df3b13b6f4ca9bbc2f8af8bceb0c3d1ecc6, hex"01");
+            totalIterations = 0;
```

Comment:
@democrazyfi this is redundant as it would correspond 1 to 1 w/ iterations on the fractal clone that we create. so we can just do IFractal(cloneAddress).iterations() to fetch this info